test/functional/page_objects/settings_page.ts

```diff
@@ -360,7 +360,7 @@ export class SettingsPageObject extends FtrService {
 
   async filterField(name: string) {
     const input = await this.testSubjects.find('indexPatternFieldFilter');
-    await input.clearValue();
+    await input.clearValueWithKeyboard();
```

Comment:
Here we like there is an issue inside the input: we have `00` there instead of `0`. I assumed this is something wrog with clearing the input and just tried to use this other way of clearing it

![image](https://user-images.githubusercontent.com/7784120/191030511-0a04618f-729e-46ab-8483-7fe0e0a646bc.png)